cmd/soroban-rpc/internal/test/simulate_transaction_test.go

```diff
@@ -771,13 +770,19 @@ func TestSimulateTransactionBumpAndRestoreFootprint(t *testing.T) {
 	assert.Greater(t, newExpirationSeq, initialExpirationSeq)
 
 	// Wait until it expires
+	expired := false
 	for i := 0; i < 50; i++ {
 		err = client.CallResult(context.Background(), "getLedgerEntry", getLedgerEntryrequest, &result)
 		if err != nil {
+			expired = true
+			fmt.Println("ledger entry expired")
 			break
 		}
+		assert.NoError(t, xdr.SafeUnmarshalBase64(result.XDR, &entry))
+		fmt.Println("waiting for ledger entry to expire at ledger", entry.MustContractData().ExpirationLedgerSeq+1)
```

Comment:
I think [`t.Logf`](https://pkg.go.dev/testing#T.Log) here and L778 would be better for test outputs

Reply:
Good idea.